### PR TITLE
[POC] Validate failure is not an error 

### PR DESCRIPTION
**Pull request by uilianries**
This PR should NOT be merged

It reproduces the error involving validate method and conan hooks.

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/foobar/0.1.0/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import os
+from conans import ConanFile, CMake
+from conans.errors import ConanInvalidConfiguration
+
+
+required_conan_version = ">=1.32.0"
+
+class FoobarConan(ConanFile):
+    name = "foobar"
```
Comment (prince-chrismc) [inline recipes/foobar/0.1.0/conanfile.py:9]: Name is too generic

```suggestion
    name = "uilianries-foobar"
```

**Review by prince-chrismc — CHANGES_REQUESTED** [review]

**Comment by SpaceIm** [comment]
> This PR should be merged

shouldn't?

**Comment by ericLemanissier** [comment]
what ? the bot reports an error on windows, which blocks the merge, whereas it should be ignored as it throws ConanInvalidConfiguration right ?

**Comment by prince-chrismc** [comment]
> Some configurations of 'foobar/0.1.0' failed in build 3 (`c11474f82befb56798774c1920bb0dca809bca31`):
> 
> * **Windows x86_64, Release, Visual Studio 14,  MT. Options:  foobar:shared-False**
>   
>   * Hooks errors detected:
>     
>     * `[HOOK - conan-center.py] pre_build(): ERROR: [FPIC MANAGEMENT (KB-H007)] 'fPIC' option not managed correctly. Please remove it for Windows configurations: del self.options.fpic (https://github.com/conan-io/conan-center-index/blob/master/docs/error_knowledge_base.md#KB-H007) `
>   * [Log download](https://c3i.jfrog.io/c3i/misc/logs/pr/4328/3/foobar/0.1.0/b3ed3b849896f7d753094dfb227a9f06b423f0c8/create_stdout.txt)
>   * [Log error Download](https://c3i.jfrog.io/c3i/misc/logs/pr/4328/3/foobar/0.1.0/b3ed3b849896f7d753094dfb227a9f06b423f0c8/create_stderr.txt)
>   * [Profile](https://c3i.jfrog.io/c3i/misc/logs/pr/4328/3/foobar/0.1.0/b3ed3b849896f7d753094dfb227a9f06b423f0c8/profile.txt)
> * [Access to all the logs](https://c3i.jfrog.io/c3i/misc/logs/pr/4328/3/foobar/0.1.0/)

This is completely incorrect from the bot

**Pull request closed without merging**